### PR TITLE
Refactor tests for functional

### PR DESCRIPTION
**Pull request by jieruan02**
This PR tries to address issue #1231.

Here is the pytest summary I got, let me know if requires further look. Also I didn't prefix the file name with functional since it's already under the functional directory, not sure if it conforms to the repo's rule or not, let me know~

====================================== warnings summary ======================================
test/torchaudio_unittest/functional/consistency_cpu_test.py::TestFunctionalFloat32::test_create_fb_matrix
test/torchaudio_unittest/functional/consistency_cpu_test.py::TestFunctionalFloat32::test_create_fb_matrix
test/torchaudio_unittest/functional/consistency_cpu_test.py::TestFunctionalFloat64::test_create_fb_matrix
test/torchaudio_unittest/functional/consistency_cpu_test.py::TestFunctionalFloat64::test_create_fb_matrix
/Users/jieruan/Library/Python/3.8/lib/python/site-packages/torchaudio/functional/functional.py:340: UserWarning: At least one mel filterbank has all zero values. The value for n_mels (10) may be set too high. Or, the value for n_freqs (100) may be set too low.
warnings.warn(

test/torchaudio_unittest/functional/consistency_cpu_test.py::TestFunctionalFloat32::test_griffinlim
/Users/jieruan/Library/Python/3.8/lib/python/site-packages/torch/functional.py:654: UserWarning: istft will require a complex-valued input tensor in a future PyTorch release. Matching the output from stft with return_complex=True. (Triggered internally at ../aten/src/ATen/native/SpectralOps.cpp:787.)
return _VF.istft(input, n_fft, hop_length, win_length, window, center, # type: ignore

-- Docs: https://docs.pytest.org/en/stable/warnings.html
======================= 103 passed, 107 skipped, 5 warnings in 28.85s ========================

### CODE REVIEW

File: test/torchaudio_unittest/functional/consistency_cpu_test.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import torch
+
+from torchaudio_unittest import common_utils
```
Comment (mthrok) [inline test/torchaudio_unittest/functional/consistency_cpu_test.py:3]: Can you change the import to

```python
from torchaudio_unittest.common_utils import PytorchTestCase
```
Same goes to `cuda_test.py`.

**Review by mthrok — CHANGES_REQUESTED** [review]
Please retain the name `torchscript_consistency_***.py`. If it's just `consistency`, then it is not immediately clear what kind of consistency it is about. (We have multiple kind of tests for checking parity.)

File: test/torchaudio_unittest/functional/consistency_cpu_test.py (hunk 1 above)
Comment (jieruan02) [reply test/torchaudio_unittest/functional/consistency_cpu_test.py:3]: no problem, I will do the both~

**Comment by jieruan02** [comment]
I kind of wonder if there is some good local tests to run before pushing. my way of testing via ci seems pretty inefficient to me...

**Review by mthrok — CHANGES_REQUESTED** [review]
Please keep the original name `torchscript_consistency`. `functional_consistency` is not clear what it means.

**Review by mthrok — CHANGES_REQUESTED** [review]
Please rename 

`test/torchaudio_unittest/functional/functional_consistency_cpu_test.py`  into `test/torchaudio_unittest/functional/torchscript_consistency_cpu_test.py` and apply the same changes to currently called `functional_consistency_*.py` scripts.

**Comment by jieruan02** [comment]
> Please rename
> 
> `test/torchaudio_unittest/functional/functional_consistency_cpu_test.py` into `test/torchaudio_unittest/functional/torchscript_consistency_cpu_test.py` and apply the same changes to currently called `functional_consistency_*.py` scripts.



> Please rename
> 
> `test/torchaudio_unittest/functional/functional_consistency_cpu_test.py` into `test/torchaudio_unittest/functional/torchscript_consistency_cpu_test.py` and apply the same changes to currently called `functional_consistency_*.py` scripts.

Oh I see, I misunderstood the part, that's why I omit the prefix at first place... Sure I will fix it now

**Comment by mthrok** [comment]
Thanks!

**Comment by jieruan02** [comment]
happy to help~
________________________________

**Pull request merged**